ansible/roles/etcd/tasks/install.yml

```diff
@@ -4,7 +4,7 @@
     etcd_source_type: "package-manager"
   when: not is_atomic and ansible_distribution != "Ubuntu" and not is_coreos
 
-- name: Install etcd
+- name: Install etcd via package manager
```

Comment:
bad commit message. First line should be understandable alone. Put the "... to support the following scenarios" in the third line.